src/SDKs/Search/DataPlane/Microsoft.Azure.Search.Data/Customizations/Documents/Models/IndexBatch.Customization.cs

```diff
@@ -9,43 +9,22 @@ namespace Microsoft.Azure.Search.Models
     using Common;
 
     /// <summary>
-    /// Contains a batch of document upload, merge, and/or delete operations to send to the Azure Search index.
+    /// Provides factory methods for creating a batch of document write operations to send to the Azure Search index.
     /// </summary>
-    public class IndexBatch : IndexBatchBase<IndexAction, Document>
+    public static class IndexBatch
```

Comment:
Looks like you have removed a number of public APIs. That is considered a breaking change, is that intended?

Reply:
Yes. This was necessary in order to decrease the footprint of custom code. Right now that's a major factor preventing us from bringing our data plane SDK to other languages. I'm working on documenting all these breaking changes. From a semver perspective, we're covered as this is a new major version.